api/commands/run.ts

```diff
@@ -27,9 +25,11 @@ export interface IExecutedAction {
 export function run(
   dbadapter: dbadapters.IDbAdapter,
   graph: dataform.IExecutionGraph,
-  partiallyExecutedRunResult: dataform.IRunResult = {}
+  partiallyExecutedRunResult: dataform.IRunResult = {},
+  runnerNotificationPeriodMillis: number = RUNNER_NOTIFICATION_PERIOD_MILLIS,
```

Comment:
rather than using these consts as defaults, use the flag values as defaults, e.g:

runnerNotificationPeriodMillis: number = flags.runnerNotificationPeriodMillis.get()

(and you can get rid of the consts entirely)